doc/manual/core_device.rst

```diff
@@ -97,7 +97,9 @@ The board has RTIO SPI buses mapped as follows:
 
 The DDS bus is on channel 32.
 
+
 This configuration supports a Zotino connected to the KC705 FMC HPC through a FMC DIO 32ch LVDS v1.2 and a VHDCI breakout board rev 1.0.
+The KC705 FMC HPC should be connected to J44A and the Zotino should be connected to J41 of the VHDCI breakout board. 
```

Comment:
The FMC breakout board does not have a "J44A" marking on its silkscreen and even less on its future front panel, so referencing it does not make the reader's life easier. You should write this documentation for someone who is setting up their boards the first time, and you should have said the *bottom* connector.